pages/common/firefox.md

```diff
@@ -27,6 +27,14 @@
 
 `firefox --headless --screenshot {{path/to/output_file.png}} {{https://example.com/}}`
 
-- Use a specific profile directory to allow multiple separate instances of Firefox to run at once:
+- Use a specific profile directory to allow multiple separate instances of Firefox to run at once (not available for Windows):
 
 `firefox --new-instance --profile {{path/to/directory}} {{https://example.com/}}`
+
+- Create a new profile in the default directory without starting Firefox:
```

Comment:
This seems the least useful to me. Most of the time you use only one profile, especially as a beginner. How about removing this to resolve the issue?

Reply:
Hrm, yeah. Firefox should create a new profile automatically if you specify the profile directory.

@zdroid: Only 1 instance of Firefox per profile may be started at any given time. To this end, if you do headless CLI Firefox stuff you usually need to specify a different directory for it to use instead.

Reply:
So to conclude, do you agree with me? I'll commit the change if you ultimately do.

Reply:
I do ultimately agree, but for different reasons, which I felt is an important distinction.